assessments/0003-krator.md

```diff
@@ -0,0 +1,175 @@
+# Krator Docs assessment
+
+## Introduction
+
+
+This is an assessment of your project documentation and website (if present), prepared for you and your project by the CNCF techdocs team.
+
+This document aims to:
+
+- Measure existing documentation quality against the CNCF’s standards
+- Recommend specific and general improvements
+- Provide examples of great documentation as reference 
+- Identify key areas which will net the largest improvement if addressed 
+
+Since the Krator documentation site is just getting started, we'll have to create or significantly develop the current content. This assessment provides a starting point for articulating current state and ideas for the future.
+
+## How this document works
+
+The assessment is divided into three sections: 
+
+- **Project documentation:** for end users of the project; aimed at people who intend to use it
+- **Contributor documentation:** for new and existing contributors to the project
+- **Website:** branding, website structure, and maintainability
+
+Within each section you receive a rating on different criteria. The full definition of each criteria is defined in [the criteria](criteria.md). We recommend reading each criterion’s definition.
+
+## Project documentation 
+
+
+| Criteria | 1 = (Is not present or requires significant work) | 2 | 3 = (is present, but needs work) | 4 | 5 = (is executed extremely well or no improvement required) |
+| ---                        | --- | --- | --- | --- | --- |
+| Information architecture   |  ✅  |     |     |    |     |
+| New user content           |  ✅  |     |     |    |     |
+| Content maintainability    |  ✅  |     |     |    |     |
+| Content creation processes |  ✅  |     |     |    |     |
```

Comment:
Because the titles in this table are so unwieldy, in the Notary assessment, I've set them up like this:
(I'm not sure this is better, but it's a little easier to work with)

```suggestion
| Criteria                   | 1   | 2   | 3   | 4   | 5   |
| ---                        | --- | --- | --- | --- | --- |
| Information architecture   | ✅  |     |     |     |     |
| New user content           | ✅  |     |     |     |     |
| Content maintainability    | ✅  |     |     |     |     |
| Content creation processes | ✅  |     |     |     |     |

 Criteria: 
 - 1 = (Is not present or requires significant work) 
 - 3 = (is present, but needs work) 
 - 5 = (is executed extremely well or no improvement required)
```